guides/source/active_record_validations.md

```diff
@@ -1448,6 +1549,12 @@ messages for a specific attribute. It returns an array of strings with all error
 messages for the given attribute, each string with one error message. If there
 are no errors related to the attribute, it returns an empty array.
 
+This method is only useful _after_ validations have been run, because it only
+inspects the errors collection and does not trigger validations itself. It's
+different from the `ActiveRecord::Base#invalid?` method explained above because
+it doesn't verify the validity of the object as a whole. It only checks to see
+whether there are errors found on an individual attribute of the object.
```

Comment:
What the pronoun 'it' refers to is not clear.
```suggestion
This method is only useful _after_ validations have been run, because it only
inspects the `errors` collection and does not trigger validations itself. It's
different from the `ActiveRecord::Base#invalid?` method explained above because
this method doesn't verify the validity of the object as a whole. `errors[]` only checks to see
whether there are errors found on an individual attribute of the object.
```